tests/test_modeling_tf_lxmert.py

```diff
@@ -751,20 +875,68 @@ def test_saved_model_creation_extended(self):
                     [num_visual_features, self.model_tester.hidden_size],
                 )
 
-                self.assertEqual(len(language_attentions), self.model_tester.num_hidden_layers["language"])
-                self.assertEqual(len(vision_attentions), self.model_tester.num_hidden_layers["vision"])
-                self.assertEqual(len(cross_encoder_attentions), self.model_tester.num_hidden_layers["cross_encoder"])
+                self.assertEqual(
+                    len(language_attentions),
+                    self.model_tester.num_hidden_layers["language"],
+                )
+                self.assertEqual(
+                    len(vision_attentions),
+                    self.model_tester.num_hidden_layers["vision"],
+                )
+                self.assertEqual(
+                    len(cross_encoder_attentions),
+                    self.model_tester.num_hidden_layers["cross_encoder"],
+                )
 
-                attentions = [language_attentions, vision_attentions, cross_encoder_attentions]
+                attentions = [
+                    language_attentions,
+                    vision_attentions,
+                    cross_encoder_attentions,
+                ]
                 attention_shapes = [
-                    [self.model_tester.num_attention_heads, encoder_seq_length, encoder_key_length],
+                    [
+                        self.model_tester.num_attention_heads,
+                        encoder_seq_length,
+                        encoder_key_length,
+                    ],
                     [
                         self.model_tester.num_attention_heads,
                         self.model_tester.num_visual_features,
                         self.model_tester.num_visual_features,
                     ],
-                    [self.model_tester.num_attention_heads, encoder_key_length, self.model_tester.num_visual_features],
+                    [
+                        self.model_tester.num_attention_heads,
+                        encoder_key_length,
+                        self.model_tester.num_visual_features,
+                    ],
```

Comment:
There's a lot of style changes in your PR - do you think you could only do the changes relative to the code you're adding? Thank you!